tests/mtouch/RegistrarTest.cs

```diff
@@ -334,6 +334,14 @@ public void MT4134 ()
 					new { Framework = "CarPlay", Version = "12.0" },
 					new { Framework = "IdentityLookupUI", Version = "12.0" },
 					new { Framework = "NaturalLanguage", Version = "12.0" },
+					new { Framework = "VisionKit", Version = "13.0" },
+					new { Framework = "SoundAnalysis", Version = "13.0" },
+					new { Framework = "PencilKit", Version = "13.0" },
+					new { Framework = "MetricKit", Version = "13.0" },
+					new { Framework = "LinkPresentation", Version = "13.0" },
+					new { Framework = "CoreHaptics", Version = "13.0" },
+					new { Framework = "BackgroundTasks", Version = "13.0" },
+					new { Framework = "QuickLookThumbnailing", Version = "13.0" },
```

Comment:
Looks good - but did this work in `xcode11`, `xcode11.1`, `xcode11.2` and `xcode11.3` branches ?

Reply:
That is a very good question, I just saw the issue starting, but I did not check on the old ones, I'll take a look. Did you see it in the last monitoring spring?

Reply:
no, that's why I'm surprised
maybe device only ?

Reply:
hmm... even if device only those all branches were merged to master so we should have seen failures there (at least)

Reply:
Here it is failing: https://github.com/xamarin/xamarin-macios/commit/c3001c8331261e4b331084dea498623ef1d2e6fa

Test results: http://xamarin-storage/jenkins/xamarin-macios/xcode11.4/c3001c8331261e4b331084dea498623ef1d2e6fa/4/jenkins-results/tests/index.html

MTouch tests:
```
Xamarin.Registrar.MT4134: errors
Expected: 15
But was: 23
```

So I'm surprised too that it did not happen in the previous spring :/

Reply:
This is the reason it happens only rarely: 

https://github.com/xamarin/xamarin-macios/pull/7931/files#diff-d231584bddcb738d5232e288f51ae8d6R288-R290

It requires Xcode 8, which isn't installed in most places.